src/com/magento/idea/magento2plugin/actions/generation/dialog/OverrideInThemeDialog.java

```diff
@@ -131,9 +131,13 @@ public static void open(final @NotNull Project project, final PsiFile psiFile) {
     }
 
     private void fillThemeOptions() {
+        final String area = psiFile.getVirtualFile().getPath().split("view/")[1].split("/")[0];
         final List<String> themeNames = new ModuleIndex(project).getEditableThemeNames();
-        for (final String themeName: themeNames) {
-            theme.addItem(themeName);
+        final String baseArea = "base";
```

Comment:
Please, add an empty line after this one.

Reply:
Done